Source/Blazorise/Components/Tabs/TabPanel.razor

```diff
@@ -1,5 +1,15 @@
 ﻿@namespace Blazorise
 @inherits BaseComponent
-<div id="@ElementId" class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
-    @ChildContent
-</div>
\ No newline at end of file
+
+@if ( Mode == TabsMode.Default )
+{
+    <div id="@ElementId" class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
+        @ChildContent
+    </div>
+}
+else if ( ( Mode == TabsMode.LazyLoad && Active ) )
```

Comment:
Can be written as `Mode == TabsMode.Default || ( Mode == TabsMode.LazyLoad && Active )` ?

Reply:
Also, I think checking for `Active` is not right. This means it will render content every time we click on a tab. The idea was to postpone rendering until we click on a tab for the first time. After that, it behaves as usual.

Reply:
Oh, we have different take on this then. I do believe it's a good feature to re-evaluate each time. Should we do a 3rd mode?

Reply:
> Can be written as Mode == TabsMode.Default || ( Mode == TabsMode.LazyLoad && Active )

😂😂 Yes

Reply:
https://github.com/Megabit/Blazorise/issues/2284#issuecomment-835825568
3rd mode for this?

Don't know what to call it yet. So we could call this version, TabsMode.Reload?
And for LazyLoad implement what you said :
> The idea was to postpone rendering until we click on a tab for the first time. After that, it behaves as usual.

Reply:
LazyReload?